docs/tutorials/replicated-stateful-application/mysql-statefulset.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1beta1
```

Comment:
+1 for having both in the same file, I like the convention and it's a bit easier to create from a single file rather than creating from two separate files.